src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,126 @@
+import cn from 'classnames';
+/* eslint-disable max-len */
+import { Filter } from '../../types/Filter';
+import { Todo } from '../../types/Todo';
+import { deleteTodos } from '../../api/todos';
+import { Errors } from '../../types/Error';
+
+interface Props {
+  clickHandler: (value: Filter) => void
+  activeFilter: Filter
+  displayTodos: Todo[]
+  setcleared: (value: boolean) => void
+  setTodos: (newTodos: Todo[] | ((prevValue: Todo[]) => Todo[])) => void
+  todos: Todo[]
+  setError: (value: string) => void
+  titleField: React.MutableRefObject<HTMLInputElement | null>;
+}
+
+export const Footer: React.FC<Props> = ({
+  clickHandler,
+  activeFilter,
+  displayTodos,
+  setcleared,
+  setTodos,
+  setError,
+  titleField,
+
+}) => {
+  const amount = displayTodos.filter(todo => !todo.completed).length;
+  const completed = displayTodos.filter(todo => todo.completed).length;
+
+  const clearCompleted = () => {
+    const toClear = displayTodos.filter((todo: Todo) => todo.completed);
+
+    toClear.forEach(todo => {
+      if (todo.id) {
+        setcleared(true);
+        deleteTodos(todo.id)
+          .then(() => {
+            setTodos((prevtodos:Todo[]) => prevtodos.filter((todoo: Todo) => todoo.id !== todo.id) as Todo[]);
+            if (titleField.current) {
+              titleField.current.focus();
+            }
```

Comment:
try this shorter version
```suggestion
              titleField.current?.focus();
```

Reply:
does not work